amida-tech/indaba-client
# Feature/inba 422 cancel project

#### What's this PR do?
Adds a go back function to the cancel button in the new project title modal
#### Related JIRA tickets:
https://jira.amida-tech.com/browse/INBA-422
#### How should this be manually tested?
Click create button
Select New Project
Click cancel
You should return to what ever screen you were on when you clicked create.
#### Any background context you want to provide?
#### Screenshots (if appropriate):
